### PR TITLE
[FIX] provisioner adapter url changes

### PR DESCRIPTION
**Pull request by sindhu-106**
Signed-off-by: sindhuja <sindhuja.gajula@intel.com>

### CODE REVIEW

File: dai_network_listener/src/main/java/com/intel/dai/network_listener/NetworkListenerConfig.java [hunk 1]
```diff
@@ -128,8 +128,8 @@ public String getFirstNetworkBaseUrl(boolean useSSL) throws ConfigIOParseExcepti
         String scheme = useSSL?"https":"http";
         PropertyMap arguments = getNetworkArguments(streamName);
         try {
-            return String.format("%s://%s:%d", scheme, arguments.getString("connectAddress"),
-                    arguments.getInt("connectPort"));
+            String hostname = arguments.getString("fullUrl").split("/")[2];
```
Comment (paul-amonson) [inline dai_network_listener/src/main/java/com/intel/dai/network_listener/NetworkListenerConfig.java:131]: Use the Url class to parse the URL, don't do it manually.

File: dai_network_listener/src/test/java/com/intel/dai/network_listener/NetworkListenerConfigTest.java [hunk 2]
```diff
@@ -362,57 +337,39 @@ public void getFirstNetworkBaseUrl() throws Exception {
...
             "    \"rackTelemetry\": {\n" +
             "      \"arguments\": {\n" +
-            "        \"connectPort\": 5678,\n" +
-            "        \"connectAddress\": \"127.0.0.1\",\n" +
-            "        \"requestBuilder\": \"foreignGetBuilder\",\n" +
-            "        \"requestType\": \"GET\",\n" +
-            "        \"urlPath\": \"/streams/rackTelemetry\"\n" +
+            "        \"fullUrl\": \"http://127.0.0.1:5678/streams/rackTelemetry\"\n" +
             "      },\n" +
             "      \"name\": \"sse\"\n" +
```
Comment (yhuan47) [inline dai_network_listener/src/test/java/com/intel/dai/network_listener/NetworkListenerConfigTest.java:350]: shall we change name from sse to eventSource?

Comment (paul-amonson) [reply dai_network_listener/src/test/java/com/intel/dai/network_listener/NetworkListenerConfigTest.java:350]: Its a test so I don't think its required. A proper unit test will NEVER create REAL network objects. :)

File: dai_network_listener/src/main/java/com/intel/dai/network_listener/NetworkListenerConfig.java [hunk 3]
```diff
@@ -128,8 +129,10 @@ public String getFirstNetworkBaseUrl(boolean useSSL) throws ConfigIOParseExcepti
         String scheme = useSSL?"https":"http";
         PropertyMap arguments = getNetworkArguments(streamName);
         try {
-            return String.format("%s://%s:%d", scheme, arguments.getString("connectAddress"),
-                    arguments.getInt("connectPort"));
+            String fullUrl = arguments.getString("fullUrl");
+            URI uri = URI.create(fullUrl);
+            return String.format("%s://%s:%d", scheme, uri.getHost(),
```
Comment (paul-amonson) [inline dai_network_listener/src/main/java/com/intel/dai/network_listener/NetworkListenerConfig.java:134]: Instead of "scheme" use uri.getScheme().

**Pull request merged**